app/src/main/res/values/strings.xml

```diff
@@ -1221,35 +1221,35 @@
     <string name="lbbs_heading">Configure 185+ blocklists</string>
 
     <!-- backup / restore bottom sheet -->
-    <string name="brbs_title">Backup &amp; restore</string>
+    <string name="brbs_title">Backup and restoration</string>
```

Comment:
`&amp;` and `restore` as before?

Reply:
Wouldn't be correct use of an ampersand.
It isn't both, but rather the concept of either(?)

Reply:
`&` is analogous to `and`: https://en.wikipedia.org/wiki/Ampersand

Reply:
Doesn't make it a direct replacement. The article says it is a substitute in "informal writing", which is the closest approximation I've seen.
Can we settle on "and"?